app/src/main/kotlin/maru/app/MaruApp.kt

```diff
@@ -61,7 +59,7 @@ class MaruApp(
       config.persistence.privateKeyPath.toString(),
     ).privateKeyBytes.toArray()
 
-  private var p2pManager: P2PManager? = null
+//  private var p2pManager: P2PManager? = null
```

Comment:
Why is the p2pManager removed, thought we needed that

Reply:
It doesn't do anything useful at the moment. I'm working to fix that in [this](https://github.com/Consensys/maru/pull/110) PR